apps/src/applab/applab.js

```diff
@@ -367,12 +367,24 @@ function shouldRenderFooter() {
   return studioApp.share;
 }
 
+const APPLAB_URL_PATTERN = /^(.*\/projects\/applab\/[A-Za-z0-9-]+)\/.*/;
```

Comment:
Note: Project keys can also contain underscores ([example](https://studio.code.org/projects/applab/7SluX0Hn_9tH8PCU-Aduvw)) which this regex doesn't handle.

Recommendation: Use character classes instead of ranges for the project key capture `[\w\d-]+`
* `\w` - Word characters, equivalent to `A-Za-z_`
* `\d` - Digits, equivalent to `0-9`